Library/Homebrew/test/language/node_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "language/node"
+
+describe Language::Node do
+  specify "#npm_cache_config" do
+    shutup do
+      ret_val = described_class.npm_cache_config
+      expect(ret_val).to eq("cache=#{HOMEBREW_CACHE}/npm_cache\n")
+    end
+  end
+
+  describe "#pack_for_installation" do
+    it "raises error with non zero exitstatus" do
+      shutup do
+        expect { described_class.pack_for_installation }.to raise_error
```

Comment:
Just an aside given I don't think this test specifically is needed: you should also use `allow_any_instance_of(Process::Status)` here or it'll be a false-negative if a user has `npm` in their `PATH`.